imagedephi/command_file.py

```diff
@@ -0,0 +1,7 @@
+from pydantic import BaseModel
+
+
+# The only two required fields are command and input_paths
+class CommandFile(BaseModel):
+    command: str
```

Comment:
The docs say that `command` is not required, but this makes `command` required.

Reply:
Changed the docs here 96476b6a47508625dee80ce4126cfac4e1cde796

Reply:
Hmm... since you might pass the a file list to plan and then to run, do we want to require `command`?

Reply:
removed in c070497166748511c94699d2af53c5dc17f850e9